README.md

```diff
@@ -204,6 +204,11 @@ We performed two change events, listed below:
     components/platform_crypto/shave_trusted_boot/xmss.md                       |  33 +++++++++
     15 files changed, 486 insertions(+), 17 deletions(-)
     ```
+  * At the time of this change, there were 1,368 lines of specification in
+    OpenSUT modules, and the change event resulted in 186 lines of
+    specification changed, reflecting 13% of the original manual effort.  See
+    [measure-change-events.sh](scripts/measure-change-events.sh) for details.
```

Comment:
Can you add the actual command you ran the script with, such that it can be reproduced in the future?